### PR TITLE
dev-docker: fix to work with current Dockerfile

### PR DESCRIPTION
**Pull request by terceiro**

### CODE REVIEW

File: test/javascript.py [hunk 1]
```diff
@@ -7,7 +7,10 @@
 
 
 def javascript_tests():
-    if os.path.exists(karma):
+    if not shutil.which('nodejs'):
```
Comment (chaws) [inline test/javascript.py]: Is there a scenario where "node_modules/.bin/karma" exists where "nodejs" doesn't? When installing npm, Debian seems to be installing nodejs too.

**Comment by chaws** [comment]
Maybe we should have javascript tests within the dev-docker? Just a suggestion...I added npm and chromium to apt in dev dockerfile then added [no-sandbox](https://github.com/karma-runner/karma-chrome-launcher/issues/189#issuecomment-440423763) to karma config and it worked fine, it just added a much greater amount of time and space

File: test/javascript.py (hunk 1 above)
Comment (terceiro) [reply test/javascript.py]: the scenario that I hit, which is I had this stuff installed in my host machine, but not in the container created by dev-docker

(I tried to add nodejs etc to it, but chromium crashes weirdly inside docker and I didn't want to investigate that at the moment)

Comment (terceiro) [reply test/javascript.py]: (just saw you other comment)

**Comment by terceiro** [comment]
@chaws so that's the trick, --no-sandbox. I will amend

**Pull request merged**